packages/lit/src/lib/Theme/index.ts

```diff
@@ -13,11 +20,20 @@ export default class Theme {
   #defaults: DesignTokens; // Every Theme has private defaults to revert to.
   #tokens: DesignTokens = {}; // In practice this will be set via the this.tokens() setter in the constructor
   #stylesheet: CSSStyleSheet = new CSSStyleSheet();
+  #errors: Map<string, ValidationError> = new Map();
```

Comment:
I don't think this should be a map, it could very well be the case that one token has multiple validationErrors, and a map would make the path unique.

I think that an array that we can filter for a specific path would be more suitable